subsys/bluetooth/host/conn.c

```diff
@@ -1912,7 +1933,54 @@ int bt_le_set_auto_conn(bt_addr_le_t *addr,
 struct bt_conn *bt_conn_create_slave_le(const bt_addr_le_t *peer,
 					const struct bt_le_adv_param *param)
 {
-	return NULL;
+	int err;
+	struct bt_conn *conn;
+	struct bt_le_adv_param param_int;
+
+	memcpy(&param_int, param, sizeof(*param));
```

Comment:
sizeof(param_int), use destination's size. I know the size is same, but good to follow a convention.